lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -51,10 +51,37 @@
       }).insertAfter(textarea);
       textarea.css('display', 'none');
 
+      // Create a div for the autosave status message
+      var autoSaveMessage = document.createElement('div');
+      $(autoSaveMessage).addClass('div position-fixed bottom-0 left-0');
+      $('body').append(autoSaveMessage);
+
+      var autoSaveTimer = null;
+
       // NOTE: This requires the page to have only one 'gollum-editor-body'.
       var editor = ace.edit(editDiv[0], {rtlText: true});
```

Comment:
@dometto Any way to get the current rtlText setting, rather than hardcoding it to true here?